pkg/secrets/check_rights_windows.go

```diff
@@ -189,6 +190,12 @@ var getDDAgentUserSID = func() (*windows.SID, error) {
 		return nil, fmt.Errorf("could not read installedDomain in registry: %s", err)
 	}
 
+	if strings.EqualFold(domain, "NT AUTHORITY") {
+		// Installer writes `NT AUTHORITY` as installedDomain which prevents the username from being resolved correctly
+		user = domain + `\` + user
+		domain = ""
+	}
+
```

Comment:
The first arguments to `windows.LookupSID`  is a hostname, not the domain. The second argument should be the fully-qualified username.
https://pkg.go.dev/golang.org/x/sys/windows#LookupSID

Reply:
Thank you @clarkb7 ! Looks like I should just:
```
sid, _, _, err := windows.LookupSID("", domain+`\`+user)
```